kolibri/core/assets/src/views/PrivacyInfoModal.vue

```diff
@@ -38,7 +38,7 @@
         <KExternalLink
           text="https://learningequality.org"
           href="https://learningequality.org"
-          target="_blank"
+          openInNewTab="true"
```

Comment:
For each of these, could you change it to
```suggestion
          :openInNewTab="true"
```
?

Otherwise, what is passed into the `openInNewTab` prop is the string literal `"true"` and not the Boolean value `true`.

Reply:
Done.